mapie/tests/test_metrics.py

```diff
@@ -33,7 +49,125 @@
         [False, True, False, True],
     ]
 )
-
+y_pred_set_2alphas = np.array([
```

Comment:
I think this is a corner case because:
1. All possible sizes of prediction sets are represented
2. You have only one occurence of each size fr each lambda
3. You have the same number of occurence for each lambda
Could you trie with an example which have none of this particularities ?